src/discrete-layer/models/discreteLayersStore.ts

```diff
@@ -33,25 +32,39 @@ export const discreteLayersStore = types
   .actions((self) => {
     const getLayersImages: () => Promise<void> = flow(
       function* getLayersImages(): Generator<
-        Promise<SearchResponse>,
+        Promise<LayersImagesResponse>, //SearchResponse
         void,
         LayersImagesResponse
       > {
         try {
           console.log('Fetch layers images--->');
           self.state = ResponseState.IDLE;
-          const result = yield self.root.fetch('/searchLayerImages', 'GET', {});
+          // const result = yield self.root.fetch('/searchLayerImages', 'GET', {});
```

Comment:
add todo to return this when search server is available

Reply:
Done